data/json/items/generic.json

```diff
@@ -488,6 +488,22 @@
     "symbol": ",",
     "color": "white"
   },
+  {
+    "id": "bundle_nylon",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": { "str": "bundle of synthetic fabric", "str_pl": "bundles of synthetic fabric" },
+    "description": "Fifty synthetic fabric patches, bundled tightly together for storage.  Disassemble to unpack.",
+    "//": "Weight is 50 x 5750mg per patch",
+    "weight": "288 g",
```

Comment:
That equals to 287.5g, not 288, making your density test fail. Do that in milligrams and it'll be fine.

Reply:
"Nobody will notice if I round this by half-a-gram, just for the thread..."  :)

Updated commit uploaded. Fingers crossed the density test will smile upon me.